core/store/src/db.rs

```diff
@@ -46,25 +46,72 @@ impl Into<io::Error> for DBError {
+// TODO(mm-near): add info about the RC in the columns.
 #[derive(PartialEq, Debug, Copy, Clone, EnumIter, BorshDeserialize, BorshSerialize, Hash, Eq)]
 pub enum DBCol {
     /// Column to indicate which version of database this is.
+    /// - *Rows*: single row [VERSION_KEY]
+    /// - *Content type*: The version of the database (u32), serialized as JSON.
     ColDbVersion = 0,
+    /// Column that store Misc cells.
+    /// - *Rows*: multiple, for example "GENESIS_JSON_HASH", "HEAD_KEY", [LATEST_KNOWN_KEY] etc.
+    /// - *Content type*: cell specific.
     ColBlockMisc = 1,
+    /// Column that stores Block content.
+    /// - *Rows*: block hash (CryptHash)
+    /// - *Content type*: [near_primitives::block::Block]
     ColBlock = 2,
+    /// Column that stores Block headers.
+    /// - *Rows*: block hash (CryptoHash)
+    /// - *Content type*: [near_primitives::block_header::BlockHeader]
     ColBlockHeader = 3,
+    /// Column that stores mapping from block height to block hash.
+    /// - *Rows*: height (u64)
+    /// - *Content type*: block hash (CryptoHash)
     ColBlockHeight = 4,
+    /// TODO: this is something related to garbage collection - no idea yet.
     ColState = 5,
+    /// Mapping from BlockChunk to ChunkExtra
+    /// - *Rows*: BlockChunk (block hash, shard id)
+    /// - *Content type*: [near_primitives::types::ChunkExtra]
     ColChunkExtra = 6,
+    /// Mapping from transaction outcome id (CryptoHash) to list of outcome ids with proofs.
+    /// - *Rows*: outcome id (CryptoHash)
+    /// - *Content type*: Vec of [near_primitives::transactions::ExecutionOutcomeWithIdAndProof]
     ColTransactionResult = 7,
+    /// Mapping from Block + Shard to list of outgoing receipts.
+    /// - *Rows*: block + shard
+    /// - *Content type*: Vec of [near_primitives::receipt::Receipt]
     ColOutgoingReceipts = 8,
+    /// Mapping from Block + Shard to list of incoming receipt proofs.
+    /// Each proof might prove multiple receipts.
+    /// - *Rows*: (block, shard)
+    /// - *Content type*: Vec of [near_primitives::sharding::ReceiptProof]
     ColIncomingReceipts = 9,
+    /// Info about the peers that we are connected to. Mapping from peer_id to KnownPeerState.
+    /// - *Rows*: peer_id (PublicKey)
+    /// - *Content type*: [network_primitives::types::KnownPeerState]
     ColPeers = 10,
+    /// Mapping from EpochId to EpochInfo
+    /// - *Rows*: EpochId (CryptoHash)
+    /// - *Content type*: [near_primitives::epoch_manager::EpochInfo]
     ColEpochInfo = 11,
+    /// Mapping from BlockHash to BlockInfo
+    /// - *Rows*: BlockHash (CryptoHash)
+    /// - *Content type*: [near_primitives::epoch_manager::BlockInfo]
     ColBlockInfo = 12,
+    /// Mapping from ChunkHash to ShardChunk.
+    /// - *Rows*: ChunkHash (CryptoHash)
+    /// - *Content type*: [near_primitives::sharding::ShardChunk]
     ColChunks = 13,
+    /// Storage for  PartialEncodedChunkV1 (seems that only for version V1 and not for V2).
```

Comment:
where is V2 stored then?

Reply:
I misread the code - seems that both versions are stored here.